src/blocks/menu-food/scss/style.scss

```diff
@@ -7,7 +7,6 @@
 
 .nova-food-menu-item {
 	display: grid;
-	align-items: baseline;
 	grid-template-columns: 1fr auto;
 	grid-template-areas:
```

Comment:
Is there any reason not to have `grid-template-areas` set to 
```
			'label label'
			'title prices'
			'description description';
```
from the start and not needing to overwrite it for the highlighted state?

Reply:
Not really, thank for the suggestion.